gpii/node_modules/preferencesServer/test/preferencesServerTests.js

```diff
@@ -580,11 +580,11 @@ var getFixtures = [
                 "gpii-default": {
                     "name": "Default preferences",
                     "preferences": {
-                        "http://registry.gpii.net/common/onscreenKeyboard": true,
                         "http://registry.gpii.net/common/initDelay": 0.120,
                         "http://registry.gpii.net/common/cursorSpeed": 0.850,
                         "http://registry.gpii.net/common/cursorAcceleration": 0.800,
                         "http://registry.gpii.net/common/mouseEmulationEnabled": true,
+                        "http://registry.gpii.net/common/onScreenKeyboardEnabled": true,
```

Comment:
@javihernandez I wasn't able to reproduce this failure - only in the case of the below array - which does seem to make sense as the ordering matter in that case (that array btw, is autogenerated based on the "*" entry in the metadata section of the NP set